Sources/Widgets/Widgets3D/SplineWidget/state.js

```diff
@@ -8,7 +8,7 @@ export default function generateState() {
   return vtkStateBuilder
     .createBuilder()
     .addField({ name: 'splineKind', initialValue: splineKind.KOCHANEK_SPLINE })
-    .addField({ name: 'splineClose', initialValue: true })
+    .addField({ name: 'isSplineClosed', initialValue: true })
```

Comment:
suggestion: splineClosed

Reply:
As it is a boolean, I think it is more logical to call it isSplineClosed.